.github/workflows/verify_consumer_pacts.yml

```diff
@@ -203,26 +203,18 @@ jobs:
         id: verification-test
         env:
           PACT_BROKER_URL: pact-broker.dsp-eng-tools.broadinstitute.org
-          PACT_PROVIDER_COMMIT: ${{ env.PROVIDER_SHA }}
-          PACT_PROVIDER_BRANCH: ${{ env.PROVIDER_BRANCH }}
-          PACT_BROKER_USERNAME: ${{ secrets.PACT_BROKER_USERNAME }}
-          PACT_BROKER_PASSWORD: ${{ secrets.PACT_BROKER_PASSWORD }}
           PACT_PROVIDER_VERSION: ${{ env.PROVIDER_VERSION }}
+          PACT_PROVIDER_BRANCH:  ${{ env.PROVIDER_BRANCH }}
+          PACT_BROKER_USERNAME:  ${{ secrets.PACT_BROKER_USERNAME }}
+          PACT_BROKER_PASSWORD:  ${{ secrets.PACT_BROKER_PASSWORD }}
         run: |
-          echo "provider-sha=${{ env.PROVIDER_SHA }}" >> $GITHUB_OUTPUT
           echo "provider-version=${{ env.PACT_PROVIDER_VERSION }}" >> $GITHUB_OUTPUT
-          echo "env.CHECKOUT_BRANCH=${{ env.CHECKOUT_BRANCH }} # If not empty, this reflects the branch being checked out (generated by Pact Broker)"
-          echo "env.CHECKOUT_SHA=${{ env.CHECKOUT_SHA }}       # If not empty, this reflects the git commit hash of the branch being checked out (generated by Pact Broker)"
-          echo "env.CURRENT_BRANCH=${{ env.CURRENT_BRANCH }}   # This reflects the branch being checked out if CHECKOUT_BRANCH is empty"
-          echo "env.CURRENT_SHA=${{ env.CURRENT_SHA }}         # This reflects the git commit hash of the branch being checked out if CHECKOUT_BRANCH is empty"
-          echo "env.PROVIDER_BRANCH=${{ env.PROVIDER_BRANCH }} # This reflects the provider branch for pact verification"
-          echo "env.PROVIDER_SHA=${{ env.PROVIDER_SHA }}       # This reflects the provider version for pact verification"
-          echo "env.CONSUMER_BRANCH=${{ env.CONSUMER_BRANCH }} # This reflects the consumer branch for pact verification (generated by Pact Broker)"
-          echo "env.CONSUMER_SHA=${{ env.CONSUMER_SHA }}       # This reflects the consumer version for pact verification (generated by Pact Broker)"
+          echo "env.PROVIDER_BRANCH=${{ env.PACT_PROVIDER_BRANCH }} # This reflects the provider branch to switch to for pact verification"
```

Comment:
Just pattern matching, but should this be `env.PROVIDER_BRANCH`, to be consistent with the `CONSUMER_{BRANCH,VERSION}` following it, and also the use on line 207?

Reply:
Changed `env.PACT_PROVIDER_BRANCH` to `env.PROVIDER_BRANCH` to be consistent with other naming.